rebasehelper/checkers/abipkgdiff_tool.py

```diff
@@ -144,17 +146,23 @@ def parse_changes(lines):
             (?P<what>Added|Changed|Removed)(\s+functions|variables)?
             (\s+\((?P<filtered_out>\d+)\s+filtered\s+out\))?
             ''', re.VERBOSE)
-            result = {}
+            result_list = []
             for line in lines:
+                tt = title_re.match(line)
```

Comment:
Why `tt`? What does it stand for? Why not `mt` (match title), to be consistent with `ms` and `mc`?